tests/debugger/probes/probe_snapshot_log_line.json

```diff
@@ -12,5 +12,4 @@
             ]
         }
     }
-    
 ]
```

Comment:
Missing newline

Reply:
That's fine. Removed redundant line.
Or do you mean something else?